src/com/rcv/Tabulator.java

```diff
@@ -327,6 +327,9 @@ private CandidateStatus getCandidateStatus(String candidate) {
       status = CandidateStatus.WINNER;
     } else if (candidateToRoundEliminated.containsKey(candidate)) {
       status = CandidateStatus.ELIMINATED;
+    } else if (candidate.equals(config.getOvervoteLabel()) || candidate
```

Comment:
Code reformatter doesn't like breaking lines on dots, recommends the following on my end:

    } else if (candidate.equals(config.getOvervoteLabel())
        || candidate.equals(config.getUndervoteLabel())) {

Reply:
Sigh. The reformatter actually made it this way on my end to resolve a line of excessive length. I guess it's not totally consistent in its choices. :/

Reply:
Yikes! If you go to Settings > Editor > Code Style > Java > Wrapping and Braces, do you have Hard wrap at 100? That should be the Google style default.

Reply:
Yes, I have that enabled.

So I just tested this and discovered something super annoying: the commit hook reformatting doesn't behave the same way that the manual Code > Reformat Code command does. If I start with the excessively long line and use the commit hook, I end up with what you saw in the initial version of this PR. If I instead do the manual command (either before or after the commit hook), it updates it to what you suggested.

Reply:
Another minor thing I've encountered: when I open the push dialog, it always defaults to origin:master, which is almost never what I want. Is there a way to get it to default to a branch of the same name as my local branch (and create that if it doesn't exist yet)? Isn't that a much more common workflow?

Reply:
Emailed response!